contracts/contracts/mcms/mcms.tolk

```diff
@@ -1341,6 +1329,9 @@ fun MCMS<T>.onInternalMessage(mutate self, msgSender: address, msgValue: coins,
             val _msgHandled = 0
                 || self.data.ownable.onInternalMessage(msgSender, msgBody);
 
+            // Notice: allow empty message (top up)
+            // @dev Contract might receive/hold TON as part of the maintenance process.
```

Comment:
curious what's the top up message (empty cell now) for ? previous when top-up msg received there's no action. What distinguish a top-up msg vs an empty msg